src/win/tty.c

```diff
@@ -386,16 +417,18 @@ int uv_tty_set_mode(uv_tty_t* tty, uv_tty_mode_t mode) {
   }
 
   uv_sem_wait(&uv_tty_output_lock);
-  if (!SetConsoleMode(tty->handle, flags)) {
+  if (
+    !SetConsoleMode(tty->handle, flags | try_set_flags) &&
+    !SetConsoleMode(tty->handle, flags)
+  ) {
```

Comment:
It's a minor thing but this indentation style is unidiomatic inside libuv.

Reply:
Done in c785048904b99f42f873c861cf556aa592394d84 👍